frontend/src/components/EditableField.js

```diff
@@ -53,11 +53,10 @@ class EditableFieldBase extends React.Component<{
 
         this.setState({ editing: false });
         event.preventDefault();
-
         $.ajax({
             type: this.props.method,
             url: this.props.url,
-            data: JSON.stringify(this.props.buildPayload(this.state.value)),
+            data: JSON.stringify(this.props.buildPayload(encodeURI(this.state.value))),
```

Comment:
These are not necessarily URIs though, right?

Reply:
Yeh, should actually be using encodeURIComponent, which is safe to use on strings